libtelemetry/src/main/java/com/mapbox/android/telemetry/BlacklistClient.java

```diff
@@ -0,0 +1,32 @@
+package com.mapbox.android.telemetry;
+
+import android.util.Log;
+
+import okhttp3.Callback;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+
+class BlacklistClient {
```

Comment:
what's the benefit of wrapping a network request into a separate class while okhttp classes are still leaked into its owner?